src/app/series/directives/series-basic.component.spec.ts

```diff
@@ -1,22 +1,44 @@
-import { cit, create, provide, By } from '../../../testing';
+import { cit, create, provide, By, cms } from '../../../testing';
+import { RouterStub, ActivatedRouteStub } from '../../../testing/stub.router';
+import { ActivatedRoute, Router } from '@angular/router';
 import { SeriesBasicComponent } from './series-basic.component';
+import { SeriesComponent } from '../series.component';
+import { ModalService, ToastrService, CmsService } from '../../core';
 import { TabService } from '../../shared';
 
+let activatedRoute = new ActivatedRouteStub();
+let router = new RouterStub();
+
 describe('SeriesBasicComponent', () => {
 
   create(SeriesBasicComponent);
 
   provide(TabService);
+  provide(Router, router);
+  provide(ActivatedRoute, activatedRoute);
+  provide(ModalService);
+  provide(ToastrService, {success: () => {}});
+  provide(SeriesComponent);
+
+  beforeEach(() => {
+    let auth = cms.mock('prx:authorization', {});
+    auth.mockItems('prx:accounts', [
```

Comment:
I believe the series.component test `defaults new series to the default account` covers the default to default account. I added a test to look for the account field on new series.